deployer/src/helpers.py

```diff
@@ -66,7 +66,7 @@ def make_request(endpoint, type=None, data=None, username=None, password=None,
                               auth=(username, password),
                               data=data)
 
-        if r.status_code / 100 != 2:
+        if r.status_code // 100 != 2:
```

Comment:
this is not very elegant IMO
```suggestion
        if 200 <= r.status_code < 300:
```

Reply:
and you should create a `is_2xx_exit_code` as you do this everywhere

Reply:
Yep thanks for sharing. Will keep it in our backlog when we will stop supporting 2.7. Thanks for sharing.